skops/card/_model_card.py

```diff
@@ -417,212 +885,222 @@ def add_table(self, folded: bool = False, **kwargs: dict["str", list[Any]]) -> C
-                card_data=self.metadata,
-                hyperparameter_table=self._extract_estimator_config(),
-                model_plot=model_plot,
-                **template_sections,
-            )
-        return card
-
-    def save(self, path: str | Path) -> None:
-        """Save the model card.
-
-        This method renders the model card in markdown format and then saves it
-        as the specified file.
-
-        Parameters
-        ----------
-        path: str, or Path
-            Filepath to save your card.
+            raise ValueError(msg)
 
-        Notes
-        -----
-        The keys in model card metadata can be seen `here
-        <https://huggingface.co/docs/hub/models-cards#model-card-metadata>`__.
-        """
-        card = self._generate_card()
-        card.save(path)
+        if self._metrics:
+            data_transposed = zip(*self._metrics.items())  # make column oriented
+            inp = {key: val for key, val in zip(["Metric", "Value"], data_transposed)}
+            table = TableSection(inp).format()
+        else:
+            # create empty table
+            table = TableSection({"Metric": [], "Value": []}).format()
 
-    def render(self) -> str:
-        """Render the final model card as a string.
+        template = textwrap.dedent(
+            """        You can find the details about evaluation process and the evaluation results.
 
-        Returns
-        -------
-        card : str
-            The rendered model card with all placeholders filled and all extra
-            sections inserted.
-        """
-        card = self._generate_card()
-        return str(card)
 
-    def _extract_estimator_config(self) -> str:
-        """Extracts estimator hyperparameters and renders them into a vertical table.
 
-        Returns
-        -------
-        str:
-            Markdown table of hyperparameters.
-        """
-        hyperparameter_dict = self.get_model().get_params(deep=True)
-        return _clean_table(
-            tabulate(
-                list(hyperparameter_dict.items()),
-                headers=["Hyperparameter", "Value"],
-                tablefmt="github",
-            )
+        {}"""
         )
+        if self.template == Templates.skops.value:
+            section = "Model description/Evaluation Results"
+        else:
+            # should be unreachable
+            msg = _make_invalid_template_error_msg(self.template)
+            raise ValueError(msg)
+
+        self._add_single(section, template.format(table))
+
+    def _generate_metadata(self, metadata: ModelCardData) -> Iterator[str]:
+        """Yield metadata in yaml format"""
+        for key, val in metadata.to_dict().items() if metadata else {}:
+            yield aRepr.repr(f"metadata.{key}={val},").strip('"').strip("'")
+
+    def _generate_content(
+        self, data: dict[str, Section], depth: int = 1
+    ) -> Iterator[str]:
+        """Yield title and (formatted) contents"""
```

Comment:
```suggestion
        """Yield title and (formatted) contents.
        
        It recursively goes through the data and yields the title
        with the appropriate number of `#`s and the associated
        content.
        """
```

Reply:
I added something similar, changed the wording a bit.